index.js

```diff
@@ -1 +1,22 @@
 console.log("Code your solution!")
+
+const input = document.querySelector('input')
+const button = document.querySelector('button')
+const form = document.querySelector('form')
+
+form.addEventListener('submit', event => {
+    event.preventDefault()
+    console.log(input.value)
+    const li = document.createElement('li')
+    li.textContent = input.value
+    document.querySelector('ul').append(li)
+
+    const list = document.querySelectorAll('ul li')
+
+    list.forEach(li => {
```

Comment:
Although it works, we don't need to loop through each `li` each time an `li` is created, adding a new event listener to each one. Instead, we could add the event listener _just_ to the one we created. Which variable holds the `li` we just created?